textattack/transformations/word_swap_contract.py

```diff
@@ -0,0 +1,94 @@
+from textattack.transformations import Transformation
+
+
+class WordSwapContract(Transformation):
+    """Transforms an input by performing contraction on recognized
+    combinations."""
+
+    reverse_contraction_map = {
+        "is not": "isn't",
+        "are not": "aren't",
+        "cannot": "can't",
+        "could not": "couldn't",
+        "did not": "didn't",
+        "does not": "doesn't",
+        "do not": "don't",
+        "had not": "hadn't",
+        "has not": "hasn't",
+        "have not": "haven't",
+        "he is": "he's",
+        "how did": "how'd",
+        "how is": "how's",
+        "I would": "I'd",
+        "I will": "I'll",
+        "I am": "I'm",
+        "i would": "i'd",
+        "i will": "i'll",
+        "i am": "i'm",
+        "it would": "it'd",
+        "it will": "it'll",
+        "it is": "it's",
+        "might not": "mightn't",
+        "must not": "mustn't",
+        "need not": "needn't",
+        "ought not": "oughtn't",
+        "shall not": "shan't",
+        "she would": "she'd",
+        "she will": "she'll",
+        "she is": "she's",
+        "should not": "shouldn't",
+        "that would": "that'd",
+        "that is": "that's",
+        "there would": "there'd",
+        "there is": "there's",
+        "they would": "they'd",
+        "they will": "they'll",
+        "they are": "they're",
+        "was not": "wasn't",
+        "we would": "we'd",
+        "we will": "we'll",
+        "we are": "we're",
+        "were not": "weren't",
+        "what are": "what're",
+        "what is": "what's",
+        "when is": "when's",
+        "where did": "where'd",
+        "where is": "where's",
+        "who will": "who'll",
+        "who is": "who's",
+        "who have": "who've",
+        "why is": "why's",
+        "will not": "won't",
+        "would not": "wouldn't",
+        "you would": "you'd",
+        "you will": "you'll",
+        "you are": "you're",
+    }
```

Comment:
since this is an exact reversal of contraction map, I think you should move contraction map to the `data.py` file and then define an __init__ function here that does something like:
```
self.reverse_contraction_map = {v: k for k, v in CONTRACTION_MAP.items()}
```
(where CONTRACTION_MAP is imported from the data file)

That should automatically reverse the contraction map from data.py. Then if we add/remove a contraction it'll automatically update this reverse contraction map too!